sparkmagic/sparkmagic/livyclientlib/sparkcommand.py

```diff
@@ -0,0 +1,120 @@
+from sparkmagic.utils.utils import coerce_pandas_df_to_numeric_datetime, records_to_dataframe
+import sparkmagic.utils.configuration as conf
+import sparkmagic.utils.constants as constants
+from sparkmagic.utils.sparkevents import SparkEvents
+from sparkmagic.livyclientlib.command import Command
+from sparkmagic.livyclientlib.exceptions import DataFrameParseException, BadUserDataException
+
+import ast
+
+class SparkStoreCommand(Command):
```

Comment:
We have discussed offline about this. There are a couple of ways to go. The straight-forward is to refactor so that the new class and SQLQuery inherit from some hybrid. 

However. if we include Command, they are all 'commands', and all (directly or indirectly) inherit from ObjectWithGuid. Hence, one can argue that SQLQuery should inherit from Command.